src/pages/student/main/PasswordReset/index.module.css

```diff
@@ -76,18 +76,32 @@
 }
 
 #Btncolor {
-  background-color: #C0D5D8;
-  border-color: #C0D5D8;
+  background-color: #c0d5d8;
+  border-color: #c0d5d8;
```

Comment:
Just a confirmation, do we not have global variables for colors?

Reply:
We don't have global variables for colors, Ms @inahsanjuan.